docs/DeveloperGuide.md

```diff
@@ -147,10 +147,10 @@ This section describes some noteworthy details on how certain features are imple
 ##### General Design
 **`Person`** component: 
 
-In Athena, contacts are represented by `Person` objects. `Person` objects have several properties such as email, address, etc. However, a `Person` can also be tagged with multiple `Tag`s.
+In Athena, contacts are represented by `Person` objects. `Person` objects have several properties such as email, address, etc. A `Person` can also be tagged with multiple `Tag`s.
 - `AddressBook` handles all direct matters concerning `Person` objects. It has a `TagManager` and `UniquePersonList`.  
-- `UniquePersonList` keeps track of all `Person` objects. Using `Person` class' `isSamePerson(Person)` method, it ensures that there are no duplicate contacts in Athena.
-- `TagManager` keeps track of which contacts contain which tags. It uses a hash map, with `Tag`s as keys and stores a set of `Person`s as its value.
+- `UniquePersonList` keeps track of all `Person` objects. It uses `Person` class' `isSamePerson(Person)` method to ensure that there are no duplicate contacts in Athena.
+- `TagManager` keeps track of which contacts contain which tags. It uses a hash map, mapping each `Tag` to the set of `Person`s that contain the `Tag`. 
```

Comment:
to _a_ set?

Reply:
I believe it's "the" set. I thought about this and settled for "the". "a" seems to imply that there exists multiple sets containing different number of `Person` objects that have the tag.